README.md

```diff
@@ -1 +1,1369 @@
-Example ForgeRock Platform Scripts
+
+# <a id="top"></a>Dynamic Scripting in ForgeRock Platform
+
+Three of ForgeRock Identity Platform products, [Access Management](https://www.forgerock.com/platform/access-management) (AM), [Identity Management](https://www.forgerock.com/platform/identity-management) (IDM), and [Identity Gateway](https://www.forgerock.com/platform/identity-gateway) (IG), allow to extend their functionality with dynamically applied, context-specific scripts written in JavaScript and Groovy and evaluated during the run time.
+
+Scripting is broadly used in the products and broadly covered across [ForgeRock Product Documentation](https://backstage.forgerock.com/docs/). There are many articles describing scripting environment and application, often in a context of particular task and supplied with examples. Other places in the documentation cover functionality that is not directly related to scripting, but can be employed by scripts.
+
+This writing aims at a quick introduction to scripting across the Platform—via comparing the scripting environments, references to the official documentation, and an example of scripting in each of the three products.
+
+## Contents
+
+* [Conclusion](#conclusion)
+* [Summary](#summary)
+* [A Simple Example of Scripting in ForgeRock Components](#example)
+    * [AM](#example-am)
+    * [IDM](#example-idm)
+    * [IG](#example-ig)
+* [References](#references)
+
+## <a id="conclusion"></a>Conclusion
+
+[Back to the Top](#top)
+
+The scripting objectives and implementation are driven by the product's functionality and the environment it provides. Hence, the scripts' location, configuration, security, the data and methods a script can use, and the way the scripts are managed are specific to a product.
+
+There are certain similarities as well: the choice of scripting languages, ability to access the underlying Java functionality and the context data, logging methods, access to the request object, and ability to make back-channel HTTP requests—all converge into a similar experience at certain level. The script configuration and content for all three products could be represented as JSON and saved in the file system.
```

Comment:
"could be represented as JSON and saved in the file system." - AM can't do that

Reply:
It is not clear here. I meant I can export scripts (in forgeops) into docker/6.5/amster/config/realms/root/Scripts.

Reply:
But I removed it from the Conclusion/Abstract section.